File: Sismique/ClientApp/app/components/shared/bridgecategory.ts
export interface IBridgeCategoryProperties {
    id: number;
    name: string;
    level: number;
}

export class BridgeCategory {
    id: number;
    name: string;
    level: number;

    deserialize(input: IBridgeCategoryProperties): BridgeCategory {
        this.id = input.id;
        this.name = input.name;
        this.level = input.level;
        return this;
    }
}

File: Sismique/Models/BridgeCategory.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ARIO.Models
{
    public class BridgeCategory
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        [Required] public string ClassifierName { get; set; }
        [Required] public string Name { get; set; }
        [Required] public int Level { get; set; }
    }
}
